packages/devkit/src/generators/project-configuration.ts

```diff
@@ -205,6 +205,22 @@ export function readProjectConfiguration(
   return getProjectConfiguration(host, projectName, workspace, nxJson);
 }
 
+/**
+ * Returns if a project has a standalone configuration (project.json).
+ *
+ * @param host - the file system tree
+ * @param project - the project name
+ */
+export function isStandaloneProject(host: Tree, project: string): boolean {
```

Comment:
Can we remove this and use `getWorkspaceLayout(host).standaloneAsDefault` instead?